recipes-kernel/linux/linux-iot2050_5.x-ti-kernel.bb

```diff
@@ -0,0 +1,34 @@
+#
```

Comment:
Do we know the exact kernel version? Better to use that exact version instead of the blurry 5.x.

Reply:
Also better to keep the same naming schema with the current `linux-iot2050_4.19.94+.bb`